### ISSUE TITLE
Will this plugin ever work for Redmine issues?

### ISSUE THREAD

**Issue by urinal-cake** [jongha/redmine_vote#7]
It's not very useful for my team, to be used on Forums only.

**Comment by biaji** [comment]
Same here.

Vote for issue  will be great!

**Comment by kfranckiewicz** [comment]
I also agree that will be very useful functionality. I want mentions also about using filters for this field/column on the issues list.
Vote! :+1:

**Comment by johnpankowicz** [comment]
I also agree that voting on issues would be great, especially for features. Voting on features is probably the best way to decide which features should be implemented.
